roofit/roofitcore/src/RooFFTConvPdf.cxx

```diff
@@ -723,52 +689,45 @@ Double_t*  RooFFTConvPdf::scanPdf(RooRealVar& obs, RooAbsPdf& pdf, const RooData
 RooArgSet* RooFFTConvPdf::actualObservables(const RooArgSet& nset) const 
 {
   // Get complete list of observables 
-  RooArgSet* obs1 = _pdf1.arg().getObservables(nset) ;
-  RooArgSet* obs2 = _pdf2.arg().getObservables(nset) ;
-  obs1->add(*obs2,kTRUE) ;
+  auto obs1 = new RooArgSet{};
+  RooArgSet obs2;
+  _pdf1.arg().getObservables(&nset, obs1) ;
```

Comment:
uuh how does this compile? shouldn't it be `*obs1` as `obs1` is a pointer and `getObservables` takes a reference as second argument? (is this calling the overload that takes a `bool` as second parameter? :eyes: )

Reply:
I HATE this implicit conversion from pointer to bool :( Thank you so much for spotting this!

Reply:
That's what reviews are for! :)